.github/ISSUE_TEMPLATE/Standard-Investigation-Issue-Template.md

```diff
@@ -0,0 +1,39 @@
+---
+name:  Investigation Issue
+about: Use this template for all Investigations
+labels: Investigation
+---
+# Story
+## Inspiration
+< What inspired this investigation >
+
+## Value of information acquired
+< What is the potential gain from the investigation outcome (customer dissatisfaction, loss of time, money, etc. >
+
+## Notes
+< special notes or suggestions information already acquired > 
+
+## Acceptance Criteria
+- [ ] A solution has been discovered and followup issue(s) / epic have been created
+- [ ] The proposed solution has been documented on this issue for approval
+
+or
+
+- [ ] A solution has been discovered but it was deemed not worth it
+
+## Subject Domain Experts
+< list of subject domain experts for this domain. If the subject expert from the end-user perspective is not the author, an end-user expert should be listed here. The Design, Dev, and QA teams should add experts here as appropriate. >
+
+## Progress
+### Reviewed Issues - completed by SM & Team Leads
+- [ ] Inspiration and Value complete
+- [ ] Initial Acceptance Criteria Defined
+- [ ] The person who will Approve story identified
+
+### Implemented Issue - completed by Developer assigned
+- [ ] Issue scored
+- [ ] Acceptance criteria met - boxes checked
+
+### Deploy-able Issue - completed by SM
+- [ ] Issue complete and correct in all required sections
+- [ ] Issue requirements met
```

Comment:
Same question here as before. I do not see a "requirements" section, so this seems somewhat ambiguous.

Reply:
I will implement the same change. Checkbox for Issue complete and checkbox for Acceptance Criteria met.